install_config/default_imagestreams_templates.adoc

```diff
@@ -0,0 +1,222 @@
+= Default Image Streams and Templates
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+Your OpenShift installation includes a useful set of Red Hat-provided
+link:../../architecture/core_concepts/builds_and_image_streams.html#image-streams[image streams] and link:../../architecture/core_concepts/templates.html[templates] to
+make it easy for developers to create new applications. By default, the
+link:quick_install.html[quick installation] and
+link:advanced_install.html[advanced installation] methods automatically create
+these sets in the *openshift* project, which is a default project to which all
+users have view access.
+
+Use the following instructions to create the objects
+yourself. The files are installed on the file system of your master.
+
+[NOTE]
+====
+This topic is only necessary if you installed OpenShift using a method other
+than the link:quick_install.html[quick installation] or the
+link:advanced_install.html[advanced installation]. Image streams and templates
+will be automatically populated in the `openshift` project when using these
+methods.
+====
+
+[[prerequisites]]
+
+== Prerequisites
+- The link:docker_registry.html[integrated Docker registry] service must be
+deployed in your OpenShift installation.
+- You must be able to run the following CLI commands with
+link:../../architecture/additional_concepts/authorization.html#roles[*cluster-admin* privileges],
+because they operate on the default *openshift*
+link:../../architecture/core_concepts/projects_and_users.html#projects[project].
+- You must have cloned the link:https://github.com/openshift/openshift-ansible/tree/master/roles/openshift_examples/files[repository] that contains the supported imagestreams:
++
+----
+$ git clone https://github.com/openshift/openshift-ansible
+----
+
+[[creating-image-streams-for-openshift-images]]
+
+== Creating Image Streams for OpenShift Images
+The core set of image streams provide images that can be used to build
+link:../../using_images/s2i_images/nodejs.html[*Node.js*],
+link:../../using_images/s2i_images/perl.html[*Perl*],
+link:../../using_images/s2i_images/php.html[*PHP*],
+link:../../using_images/s2i_images/python.html[*Python*], and
+link:../../using_images/s2i_images/ruby.html[*Ruby*] applications. It also
+defines images for link:../../using_images/db_images/mongodb.html[*MongoDB*],
+link:../../using_images/db_images/mysql.html[*MySQL*], and
+link:../../using_images/db_images/postgresql.html[*PostgreSQL*] to support data
+storage.
+
+If your node hosts are subscribed using Red Hat Subscription Manager and you
+want to use the Red Hat Enterprise Linux (RHEL) 7 based images:
+
+----
+$ oc create -f \
+    openshift-ansible/roles/openshift_examples/files/examples/v1.1/image-streams/image-streams-rhel7.json \
+    -n openshift
+----
```

Comment:
All the JSON file paths for OSE should be updated per https://github.com/openshift/openshift-docs/issues/1454#issuecomment-173228118.